wuvt/admin/views.py

```diff
@@ -492,6 +492,27 @@ def donation_index():
         if 'reset_stats' in request.form:
             redis_conn.set('donation_stats_start',
                            datetime.datetime.utcnow().isoformat())
+        if 'set_paid' in request.form:
+            # TODO log who set as paid
```

Comment:
I don't love leaving these TODOs in the source code with nothing else to keep track of them; they're easy to forget. Can you create an issue for this one please?